pkg/component/controller/etcd.go

```diff
@@ -126,46 +163,12 @@ func (e *Etcd) Run() error {
 
 	if util.FileExists(filepath.Join(e.K0sVars.EtcdDataDir, "member", "snap", "db")) {
 		logrus.Warnf("etcd db file(s) already exist, not gonna run join process")
```

Comment:
I think instead of "else if" here we can just have explicit return if the guard check has not passed, to keep nesting level lower.